js/index.js

```diff
@@ -0,0 +1,81 @@
+var url;
+var content = document.getElementById("content");
+var search = document.getElementById("search");
+
+search.addEventListener("click", () => {
+    var searchText = document.getElementById("searchText").value;
+    url = `https://www.googleapis.com/books/v1/volumes?q=${searchText}&maxResults=40`;
+    if (searchText !== "" && searchText !== " ") {
+        getData(url);
+    } else {
+        window.alert("Please enter a search term that contains at least one alphanumeric character")
+    }
+})
+
+window.onload = () => {
+    url = "https://www.googleapis.com/books/v1/volumes?q=pride+prejudice&maxResults=40"
+    getData(url);
+}
+
+const getData = (url) => {
+    var request = new XMLHttpRequest();
+    request.open('GET', url);
+    request.onload = () => {
+        if (request.status >= 200 && request.status < 400) {
```

Comment:
The success status code will always be 200. So `if (request.status === 200)` should be a sufficient check.

Reply:
done. Thanks